.github/workflows/main.yml

```diff
@@ -1,22 +1,6 @@
 name: QA
```

Comment:
if we are on subject of minimal template - maybe it will be worth adding `.nvmrc` with latests node LTS to repo and using [Node version file](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file) in this workflow?

Reply:
Or even better: use .npmrc -> https://pnpm.io/npmrc#use-node-version
pnpm will automatically use desired version <3

Reply:
(installing and using version process is totally hidden from the user, everything `works out of the box` TM )

Reply:
@krzysztofwolski I think this is feature when you use pnpm as a node version manager

Action itself doest use it
https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file

Reply:
pnpm also respects nvmrc, so I'm pleased :D